src/renderer/pages/project/script_editor/reference.vue

```
@@ -180,4 +181,16 @@
     reader.readAsArrayBuffer(file);
   }
 };
+
+const submitImage = async (imageKey: string, key: number) => {
+  try {
+    imageFetching.value = true;
+    const res = (await window.electronAPI.mulmoHandler("mulmoReferenceImage", props.projectId, key, imageKey, {
```

ESLint

[error] 'res' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)